static/map.js

```diff
@@ -42,6 +42,7 @@ function pad(num, size) {
 document.getElementById('pokemon-checkbox').checked = getFromStorage("displayPokemons", "true");
 document.getElementById('gyms-checkbox').checked = getFromStorage("displayGyms", "true");
 document.getElementById('coverage-checkbox').checked = getFromStorage("displayCoverage", "true");
+document.getElementById('pokestops-checkbox').checked = getFromStorage("displayPokestops", true);
```

Comment:
`true` is the right expression there.
But the option pokestops inside `app.py` was set to false as default. (for case: not specified in request)

Default is now **don't** Display-pokestops.